pmg/static/resources/javascript/hansards.js

```diff
@@ -0,0 +1,19 @@
+// message if no hansards in filter
+if($(".hansards ul.hansard-list > li").length == 0)
+  $(".hansards ul.hansard-list").append("<li>No hansards found. Please expand your search.</li>");
```

Comment:
this is better done in the template than in javascript. We should try to do page content in the template wherever possible.